css/index.css

```
@@ -253,3 +254,237 @@ body {
     font-size: 14px;
   }
 }
+
+
+/* Movil style */
+.product {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+  padding: 20px;
+  margin: auto;
+}
+
+.rating {
+  display: inline-flex;
+  align-items: center;
+}
+
+.star {
+  width: 16px;
+  height: 15.4px;
+  color: #FDBF00;
+  margin-left: -13px;
+  margin-top: 8px;
+}
+
+.product__rating{  
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 17.6px;
+  margin-left: 2px;
+  margin-top: 9.5px;
+  color: #6C6B6A;
+}
+.carousel {
+  position: relative;
+}
+
+.carousel__product {
+  position: relative;
+  max-width: 773px;
+  width: 100%;
+  overflow: hidden;
+  box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.13);
+  border-radius: 5px;
+  margin: auto;
+}
+
+.carousel__img {
+  display: flex;
+  transition: transform 0.2s ease;
+}
+
+.carousel__item {
+  min-width: 100%;
+  height: 290px;
+  background-size: cover;
+  background-position: center;
+}
+
+.js-prev,
+.js-next {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  background-color: white;
+  border: 1px solid black;
+  border-radius: 50%;
+  padding: 10px;
+  cursor: pointer;
+  width: 43px;
+  height: 45px;
+}
+
+.js-prev {
+  left: 10px;
+}
+
+.js-next {
+  right: 10px;
+}
```

Como te mencione hace un rato, las clases `js-` no llevan css
```suggestion
.prev {
  left: 10px;
}

.next {
  right: 10px;
}
```